docs/content/developers/remote-config.md

```diff
@@ -0,0 +1,95 @@
+# Remote Config
+
+DDEV supports downloading a remote config from the [`ddev/remote-config`](https://github.com/ddev/remote-config)
+GitHub repository with messages that will be shown to the user. This feature
+could be enhanced later with more information and configuration.
+
+Here is an example of a `remote-config.jsonc` file:
```

Comment:
This might be better just as a link to the remote-config repo.

Reply:
Agree, on place less to maintain.